packages/augur-state/src/logs/types.ts

```diff
@@ -6,10 +6,14 @@ export interface Log {
     transactionIndex: number;
     transactionHash: string;
     logIndex: number;
-    timestamp: number;
```

Comment:
I realize the log's block timestamp isn't currently getting saved to the DB, but shouldn't it be? For example, `getMarketsInfo` is supposed to return a `creationTimestamp` for the market. Shouldn't we be saving this info to the DB and leaving `timestamp` here?

Reply:
I'm taking the approach of only including timestamp where needed. We had these in every log in Augur Node by virtue of the architecture so it just got included everywhere. For ordering by time we can just use the id / blocknumber.

Reply:
More specifically unless we have a reason to search or segment by timestamp we shouldn't actually need it.

Reply:
Ok, looking through the code more, it looks like `creationTimestamp` could probably be removed anyway from getMarketsInfo.